data/sql/updates/pending_db_world/rev_1609431870979648000.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609431870979648000');
+UPDATE `gameobject_template` SET `Data2` = 0, `AIName` = 'SmartGameObjectAI' WHERE `entry` = 181964;
+DELETE FROM `event_scripts` WHERE `id` = 11027;
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (181964, 17715, 1771500);
+INSERT INTO `smart_scripts` VALUES
+	(181964, 1, 0, 0, 70, 0, 100, 0, 2, 0, 0, 0, 0, 12, 17715, 4, 300000, 0, 0, 0, 8, 0, 0, 0, 0, -1896.13, -12861.1, 87.1412, 3.5412, 'Statue of Queen Azshara - On Gameobject State Change - Summon Creature \'Atoph the Bloodcursed\'');
+INSERT INTO `smart_scripts` VALUES
+	(17715, 0, 0, 0, 63, 0, 100, 0, 0, 0, 0, 0, 0, 80, 1771500, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Atoph the Bloodcursed - Just Summoned - Run Script');
+INSERT INTO `smart_scripts` VALUES
```

Comment:
you can just have one `INSERT INTO `smart_scripts` VALUES`, no need to have 3. See: https://www.azerothcore.org/wiki/Dealing-with-SQL-files#write-compact-code

Reply:
done